spec/views/hq/sponsors/index_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'will_paginate/array'
```

Comment:
Removing this line for me results in this error message:
`Failure/Error: items.paginate(:page => 2, :per_page => 2)
     NoMethodError:
       undefined method `paginate' for #<Array:0x0000000baa0838>`
Any idea?